packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_fr_FR.json

```diff
@@ -482,9 +482,10 @@
   "domain_configuration_dns_entry_add_step1_helpwizard_txt_desc": "Permet d’entrer n’importe quel texte dans la zone DNS.",
   "domain_configuration_dns_entry_add_step3_info": "Vous allez ajouter l'entrée suivante dans votre zone DNS :",
   "domain_configuration_dns_entry_add_step3_alert": "L’ajout sera immédiat dans la zone DNS, mais veuillez prendre en compte le temps de propagation (maximum 24h).",
+  "domain_configuration_dns_entry_add_step3_check_warning0": "Erreur : Impossible d'enregistrer la configuration DNS. Merci de vérifier que:",
```

Comment:
**nitpick:** Can you use other name apart from `warning0`?

Reply:
Done